frontend-instructor/src/components/mycourses/MyCourses.jsx

```
@@ -1,15 +1,95 @@
+                    </button>
+                  </div>
+                </div>
+              </div>
+            ))}
+          </div>
+
+          {/* Add Course Button */}
+          <div className="text-center mt-6">
+            <Link to="/add-courses">
+              <button className="px-6 py-3 bg-green-500 text-white font-bold rounded-lg hover:bg-green-600 transition">
+                + Add Course
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)